cfme/test_framework/pytest_plugin.py

```diff
@@ -103,6 +103,7 @@ def pytest_collection_finish(session):
     'cfme.fixtures.tccheck',
     'cfme.fixtures.pxe',
     'cfme.fixtures.candu',
+    'cfme.fixtures.v2v',
```

Comment:
@Yadnyawalkya I have not moved the fixtures to v2v.py yet, have you tested it to work?